### ISSUE TITLE
Implementation of priority list

### ISSUE THREAD

**Issue by andy5995** [Sw24Softwares/StarkeVerben#24]
When using the Lessons feature, it seems the words appear randomly. Sometimes there is a word I would like to see appear more often to help me remember better.

To implement this:

* On the SingleLesson page for the word, a checkbox that says "add to priority list" or something like that

* The "List" menu would change to "Lists", and show the name of the list of all the words, and the name of the priority list.

* On the "Lessons" menu, the user would have the option to be shown words from the normal list or the priority list.

**Comment by Swarthon** [comment]
Indeed, it's a problem I faced but I haven't thought about it. Your idea seems very effective. For the graphical part, @24PaH will be more qualified to answer as he managed it. Nevertheless, I do have some ideas to tweak the random pick of the verb.
Thank you for being involved in our project.
